src/main/webapp/i18n/de/exam.json

```diff
@@ -296,6 +297,42 @@
             "delete": {
                 "question": "Soll die Klausur <strong>{{ title }}</strong> wirklich dauerhaft gelöscht werden? Alle zugehörigen Elemente werden gelöscht, inklusive der Klausuren der Studenten. Diese Aktion kann NICHT rückgängig gemacht werden!",
                 "typeNameToConfirm": "Bitte gib den Namen der Klausur zur Bestätigung ein."
+            },
+            "checklist": {
+                "title": "Checkliste",
+                "introduction": "Unter folgendem Link können sie mehr über den Ablauf einer Artemis Klausur informieren:",
+                "linkDescription": "Instructors guide for exams in Artemis",
+                "checklistItem": "Checkliste",
+                "description": "Beschreibung",
+                "goTo": "Gehe zu",
+                "tableItem": {
+                    "register": "Registrieren der Studierenden zur Klausur",
+                    "exerciseGroups": "Erstellen Sie mindestens eine Aufgaben Gruppe mit mindestens einer Aufgabe",
+                    "generateStudentExams": "Generieren aller Klausuren",
+                    "prepareExerciseStart": "Aufgabenstart vorbereiten",
+                    "examDetails": "Ausfüllen aller Klausur Details",
+                    "conductExam": "Abhalten der Klausur",
+                    "assessAllSubmissions": "Alle Abgaben bewerten",
+                    "publishResults": "Veröffentlichen der Ergebnisse",
+                    "examReview": "Planung der Klausureinsicht ",
+                    "resolveComplaints": "Auflösen aller Beschwerden",
+                    "testRun": "Abhalten eines Testdurchlaufs",
+                    "exportResults": "Exportiere die Klausurergebnisse"
+                },
+                "descriptionItem": {
+                    "register": "Mindestens ein Studierender muss zur Klausur angemeldet sein.",
+                    "exerciseGroups": "Die Punkte aller Aufgaben einer Aufgabengruppe muss identisch sein. Die Anzahl an Aufgabengruppen muss der Anzahl der Aufgaben in der Klausur entsprechen.",
+                    "generateStudentExams": "Die Klausuren der Studierenden müssen generiert worden sein, bevor die Klausur beginnt.",
+                    "prepareExerciseStart": "Vorbereiten der Aufgaben kann die Serverlast während der Klausur reduzieren.",
+                    "examDetails": "Alle relevanten Details der Klausur müssen vor dem Start der Korrektur eingetragen sein.",
+                    "conductExam": "In der Studierenden-Klausurübersicht sieht man, welche Studierende ihre Klausur bereits abgegeben haben.",
+                    "assessAllSubmissions": "Die Anzahl der Korrekturrunden der Aufgaben entspricht der in der Klausur angegebenen Anzahl von Korrekturrunden.",
```

Comment:
Korrekturrunden → Korrektur-Runden
Improves readability

Reply:
to be honest, I am not a big fan of those dashes (`-`) between words. I would like to get rid of all of those ;-)

Reply:
Done